server/controllers/applicant/getApplicantsStats.js

```diff
@@ -0,0 +1,36 @@
+const Applicant = require('../../database/models/applicant');
+
+const getApplicantsStats = (req, res, next) => {
+  Applicant.find({}, (err, applicants) => {
+    if (err) {
+      return res.status(400).json({ statusCode: 400, error: 'Bad Request' });
+    }
+    if (!applicants.length) {
+      return res.status(404).json({ statusCode: 404, error: `Data not found` });
+    }
+    return true;
+  });
+  Promise.all([
+    Applicant.find({ accepted: true }).countDocuments(),
+    Applicant.find({
+      applicationSubmittedDate: { $exists: true, $ne: null },
+    }).countDocuments(),
+    Applicant.find({
+      applicationSubmittedDate: null,
+    }).countDocuments(),
+  ])
+    .then((results) => {
```

Comment:
@alidah0 If you want, you could destructure the results argument right here:
`.then(([accepted, submitted, notSubmitted]) => {`